CMakeLists.txt

```
@@ -113,8 +113,10 @@ function(packageProject)
     endif()
   endif()
 
+  set(INSTALL_DIR_FOR_CMAKE_CONFIGS ${CMAKE_INSTALL_LIBDIR})
   if(PROJECT_ARCH_INDEPENDENT)
     set(wbpvf_extra_args ARCH_INDEPENDENT)
+    set(INSTALL_DIR_FOR_CMAKE_CONFIGS ${CMAKE_INSTALL_DATADIR}) # share
   endif()
```

- Could it be more legible if we avoid overwriting variables?
- Can the comment be more verbose?

```suggestion
  if(PROJECT_ARCH_INDEPENDENT)
    set(wbpvf_extra_args ARCH_INDEPENDENT)
    # install to architecture independent (share) directory
    set(INSTALL_DIR_FOR_CMAKE_CONFIGS ${CMAKE_INSTALL_DATADIR})
  else()
    set(INSTALL_DIR_FOR_CMAKE_CONFIGS ${CMAKE_INSTALL_LIBDIR})
  endif()
```